.vscode/tasks.json

```diff
@@ -7,8 +7,9 @@
 			"options": {
 				"cwd": "${workspaceFolder}/packages/devtools_app",
 			},
-			"command": "pub",
+			"command": "flutter",
```

Comment:
shouldn't this task now be
`flutter run -d chrome --no-web-initialize-platform`
?

Reply:
No, it runs on flutter pub.  The CONTRIBUTING.md file and the integration tests reflect this too.